Changelog (#1390)
microsoft/appcenter-sdk-android: CHANGELOG.md

> ## Version 3.1.1 (Under development)

Reviewer: ```suggestion
## Version 3.2.0 (Under development)
```

Since it's labeled as feature.
Should also bump versions.gradle to use 3.2 so snapshot builds are correctly versioned as well.

### App Center Distribute

> * **[Feature]** Enable checking for update when application switches from background to foreground.

Reviewer: ```suggestion
* **[Feature]** Automatically check for update when application switches from background to foreground (unless automatic checks are disabled).
```

> * **[Feature]** Enable checking for update when application goes into background and then goes into foreground.

Reviewer 2: ```suggestion
* **[Feature]** Enable checking for update when application switches from background to foreground.
```
